config/jobs/kubernetes/sig-scheduling/sig-scheduling-config.yaml

```diff
@@ -80,7 +80,7 @@ presubmits:
         args:
         - /bin/sh
         - -c
-        - ./hack/install-etcd.sh && make test-integration WHAT=./test/integration/scheduler_perf ETCD_LOGLEVEL=warn KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-run=^$$ -benchtime=1ns -bench=BenchmarkPerfScheduling"
+        - ./hack/install-etcd.sh && PATH=${PWD}/third_party/etcd:${PATH} make test-integration WHAT=./test/integration/scheduler_perf ETCD_LOGLEVEL=warn KUBE_TEST_ARGS="-run=xxxxxx -benchtime=1ns -bench=BenchmarkPerfScheduling"
```

Comment:
xxx??

Reply:
Use something as "string-to-not-match-any-unit-test" per example, so we clearly know the goal